cats/src/main/scala/magnolify/cats/auto/CatsMacros.scala

```diff
@@ -49,18 +55,24 @@ private object CatsMacros {
 }
 
 trait LowPriorityImplicits extends LowPriorityGenGroup {
-  implicit def genEq[T]: Eq[T] = macro CatsMacros.genEqMacro[T]
```

Comment:
why is this implicit no longer there?

Reply:
I movied it further down to below `Hash` since `Hash` extends `Eq`.

Reply:
Ohh, I see now, thanks.